src/super_gradients/recipes/training_hyperparams/default_train_params.yaml

```diff
@@ -1,5 +1,6 @@
-resume: False # whether to continue training from ckpt with the same experiment name.
-resume_path: # Explicit checkpoint path (.pth file) to use to resume training.
+resume: False # Whether to continue training from ckpt from the latest run, within the same experiment name.
+run_id:       # ID of run to resume from the same experiment.
```

Comment:
Please also add this parameter documentation in the Trainer.train(..) doctoring.
Also, I think there are multiple .md documentation files that need to know about this feature.

Reply:
.md doc added ced84409863d301a0b70f0454377e9bc612b2696
Trainer doc added https://github.com/Deci-AI/super-gradients/pull/1353/commits/e46007d5140e85f7abd98db89e1e8d50946776e7